### PR TITLE
Added SweetSwift.

### PR DESCRIPTION
**Pull request by Elland**

### CODE REVIEW

File: Tests/Date+HelperTests.swift [hunk 1]
```diff
@@ -4,17 +4,17 @@ import XCTest
...
-        let lastDay = datesOfComingYear.last
+        let firstDay = dateWithNext30.first
+        let lastDay = dateWithNext30.last
 
-        let dayInAYear = Calendar.autoupdatingCurrent.date(byAdding: .day, value: 30, to: date)
+        let day31 = Calendar.autoupdatingCurrent.date(byAdding: .day, value: 30, to: date)
 
         XCTAssertEqual(date, firstDay)
-        XCTAssertEqual(dayInAYear, lastDay)
+        XCTAssertEqual(day31, lastDay)
```
Comment (designatednerd) [inline Tests/Date+HelperTests.swift]: did you mean for these test changes to go on this branch or the other PR?

Comment (Elland) [reply Tests/Date+HelperTests.swift]: Here, as I'm currently fixing these, b/c timezones are the worst. 👍

**Comment by Elland** [comment]
Got that nice ✅ back!

File: Tests/SunriseSet+HelperTests.swift [hunk 2]
```diff
@@ -8,7 +8,7 @@ class Suntimes_HelperTests: XCTestCase {
     func testDaylightLengthProgress() {
         let coordinate = CLLocationCoordinate2D(latitude: 59.917, longitude: 10.75)
 
-        let timeZone = TimeZone(identifier: "UTC")!
+        let timeZone = TimeZone(secondsFromGMT: 0)!
```
Comment (designatednerd) [inline Tests/SunriseSet+HelperTests.swift]: is this necessary because circle is running on GMT instead of UTC?

Comment (Elland) [reply Tests/SunriseSet+HelperTests.swift]: I tried using UTC, and it returns GMT. 🤦‍♀️

Comment (designatednerd) [reply Tests/SunriseSet+HelperTests.swift]: So looking at this [in a playground](https://gist.github.com/designatednerd/25f90de58ee3d492f2b55f209a0a886b), despite `UTC` showing up in `TimeZone`'s `abbreviationDictionary` as mapping from `"UTC"` to `"UTC"`, doing any of the following: 

```swift
let utc = TimeZone(identifier: "UTC")
let gmt = TimeZone(secondsFromGMT: 0)
let gmt2 = TimeZone(identifier: "GMT")
```

Prints out `GMT (fixed)` when logged. 

A clue seems to be that while `GMT` is in the list of `knownTimeZoneIdentifiers`, while `UTC` is not. But wow, that's confusing.

Comment (Elland) [reply Tests/SunriseSet+HelperTests.swift]: Right? Aren't time zones complicated enough?!

**Pull request merged**